app/src/marketplace/components/ProductPage/StreamListing/index.jsx

```diff
@@ -66,7 +64,8 @@ const HoverComponent = ({
                 outline
                 tag={Link}
                 className={styles.button}
-                href={formatExternalUrl(links.editor.canvasEditor, {
```

Comment:
Don't see a reason for this one to be external. Browser's "Copy link address" will copy over the full address anyway if one wants to share it. Dropped the external part.